README.md

```diff
@@ -1,2 +1,83 @@
+- give you timely feedback, both good and constructive
+- get you the resources you need
+- challenge you in ways that will help you grow
+- champion your accomplishments to anyone who will listen (and even those who won't)
+
+My role is not to:
+
+- do your job for you
+- enable laziness
+
+You do have to meet me halfway. I will be your biggest cheerleader, but I can't be everywhere at once! Sometimes I'll ask you to write down what your doing to help me out. Sometimes it's because I legit have too many things to focus on and I need an easy way to recall all the amazing things that you've done lately. Help me help you get more money!
+
+## What do I value most?
+
+I value honesty.
+
+I value genuine efforts to improve.
+
+**I value the safety of the marginalized over the hurt feelings of the privileged.**
+
+I value collaboration and clear communication.
+
+I value mental health over shipping velocity.
+
+I value emotional labor.
+
+I understand that it can be difficult to trust a new manager. You've probably been burned before! But hopefully laying out my values can help you understand where I'm coming from. I will always tell you as much information as I know and am able to tell you. I will always care about you as a person and your wellbeing. I will always fight for what is the right thing to do. My values are the most important thing to me.
+
+## My Expectations
+
+I expect you, dear reader, to give me your best effort each day. I understand some days are just not working out. That's totes fine. I want to work with you to make sure that you have a reasonable workload with reasonable deadlines and you are taking care of yourself. If you aren't able to give it your all for whatever reason, let's talk about it sooner rather than later!
+
+I expect you to show up on time and be present at meetings. I hate useless meetings and promise to keep them to a minimum. There will always be an agenda. They'll probably always be remote meetings. So put on some pj pants and hop on video and be fully present for the short time we have together.
+
+I expect you to help out your fellow teammates if they need help. We all have trouble here and there and no one likes to feel like a burden! If someone is asking for help then let's help teach one another the skills we need to solve the problem.
+
+I expect you to be respectful of one another's lived experiences. I hate to use a buzzword, but diversity & inclusion is super important to making good teams. That means sometimes misunderstandings happen! But I expect you to be respectful of someone's lived experiences. However, keep in mind that I value the safety of the marginalized over the hurt feelings of those with more privilege. Power dynamics in terms of race, gender and gender identity, sexual identity or expression, class, disability, religion, age, body size, etc will always be kept into consideration. Clarifying questions: good. Dismissing someone's lived experiences: bad.
```

Comment:
> gender and gender identity

Not super sure what you mean here? Possibly "gender identity and expression"?

Reply:
> kept into consideration

"taken into" or "kept in", I think. Unsure though 🤔

Reply:
> Possibly "gender identity and expression"?

Yes, that's what I meant! Should've copied and pasted from a better source than my memory